data/migrations/20190405085804_projects_table.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('projects', tbl => {
+      tbl.increments()
+      tbl.string('project_name', 128).notNullable()
+      tbl.string('description', 200).notNullable()
+      tbl.boolean('isCompleted').notNullable()
```

Comment:
`.notNullable()` is not necessary for `tbl.boolean` as it will be created automatically by Knex. You should add `.defaultTo(false)`, however.